android/src/com/frostwire/android/gui/adapters/menu/RenameFileMenuAction.java

```diff
@@ -56,7 +62,11 @@ protected void onClick(Context context) {
 
     private void showRenameFileDialog() {
         RenameFileMenuActionDialog.renameAction = this;
-        RenameFileMenuActionDialog.newInstance(getFilePath()).show(((Activity) getContext()).getFragmentManager());
+        RenameFileMenuActionDialog renameFileMenuActionDialog = RenameFileMenuActionDialog.newInstance(getFilePath());
```

Comment:
I'll make `RenameFileMenuActionDialog.newInstance()` more consistent.
Right now I'm using a setter on `RenameFileMenuActionDialog` and on `DeleteFileMenuActionDialog` I'm passing the dialog listener as part of the `newInstance` parameters.